File: 01_basics/01_09_is_prime/IsPrime.java
public class IsPrime {
    public static void main(String[] args) {
        int num = 23;
        System.out.println(String.format("Is %d a prime number: %b", num, isPrime(num)));
    }

    private static boolean isPrime(int n) {
        for (int i = 2; i < Math.sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }
}

File: 01_basics/01_09_is_prime/is_prime.js
const isPrime = (n) => {
    for (let index = 2; index < Math.sqrt(n); index++) {
        if (n % index === 0) {
            return false;
        }
    }
    return true;
}

const num = 23;
console.log(`Is ${num} a prime number: ${isPrime(num)}`);